mirdata/core.py

```diff
@@ -391,6 +446,51 @@ class MultiTrack(Track):
 
     """
 
+    def __init__(
```

Comment:
For the end of the init, should we also add:

```
self._index = index
self.track_ids = self._index["multitracks"]["tracks"]
```
and maybe two additional properties, one for getting all associated track objects:
```
@property
def tracks(self):
    return {t: Track(t, self.data_home, self.dataset_name, self._index, self._metadata) for t in self.track_ids}
```

and one defining the audio property to use when mixing, which in core we can default to None, but we can overwrite in the loaders
```
@property
def track_audio_property(self):
    raise NotImplementedError("Mixing is not possible in this dataset")
```

With that we can remove the `_check_mixable` function everywhere in the code below.